### ISSUE TITLE
Custom loss function

### ISSUE THREAD

**Issue by yug125lk** [JuliaWolleb/Diffusion-based-Segmentation#49]
Hi, thank you for sharing this impressive work! I have a question: how can I train the model using a custom loss function that compares the generated image with the ground truth? Thank you in advance.

**Comment by JuliaWolleb** [comment]
Hi

We based our diffusion model on this work[ Diffusion Models beat GANs](https://arxiv.org/abs/2105.05233). There the loss function is defined as an MSE loss between added and predicted noise. It has shown to be stable.
If you still want to change it, you can change it [here](https://github.com/JuliaWolleb/Diffusion-based-Segmentation/blob/676f214035e90edd0357f51feab45841e4aefcfb/guided_diffusion/gaussian_diffusion.py#L907 )
